RockWeb/App_Code/GetEventCalendarFeed.cs

```diff
@@ -115,70 +117,87 @@ private iCalendar CreateICalendar( CalendarProps calendarProps )
             {
                 foreach ( EventItemOccurrence occurrence in eventItem.EventItemOccurrences )
                 {
-                    iCalendarSerializer serializer = new iCalendarSerializer();
-                    iCalendarCollection ical = ( iCalendarCollection ) serializer.Deserialize( occurrence.Schedule.iCalendarContent.ToStreamReader() );
-
-                    foreach ( var icalEvent in ical[0].Events )
+                    if ( occurrence.Schedule != null )
```

Comment:
@chrismcgrath  As it currently stands, I'm having a hard time seeing what else was changed in this method.  I wonder if this was written as an 'if continue' I think it would be much easier to read the diff:

```
   if ( occurrence.Schedule == null )
   {
      continue;
   }
```

Reply:
Agreed, unfortunate that it displayed the diff this way.  I commented the other changes below.